### PR TITLE
Jira ticket # -paraLeo

### PR DESCRIPTION
**Pull request by strgleon**
added:
feature file
steps defs
fixture
POM

### CODE REVIEW

File: cypress/support/step_definitions/Predefined_Steps.js [hunk 1]
```diff
@@ -1,380 +0,0 @@
-/// <reference types="cypress" />
-import {
-  Given,
-  When,
-  Then,
-  And,
```
Comment (TroyHarding) [inline cypress/support/step_definitions/Predefined_Steps.js:6]: Remove AND from statement

**Review by TroyHarding — CHANGES_REQUESTED** [review]
Either change or the Predefiend from your review

**Review by strgleon — COMMENTED** [review]
no ANd

File: cypress/support/step_definitions/Predefined_Steps.js (hunk 1 above)
Comment (TroyHarding) [reply cypress/support/step_definitions/Predefined_Steps.js:6]: Resolved

**Review by TroyHarding — APPROVED** [review]

**Pull request merged**